development/plans/eclipse_project_plan_4_33.xml

```diff
@@ -0,0 +1,406 @@
+		<tr class="c1">
+			<td rowspan="2">Apple macOS</td>
+			<td rowspan="1">
+				12<br/>
+				13
+			</td>
+			
+			<td rowspan="1">x86 64-bit</td>
+			
+			<td rowspan="1">
+				OpenJDK 17.0.8 (LTS)<br/>
+				OpenJDK 21 (LTS)<br/>
+				Oracle Java 17.0.8 (LTS)<br/>
+				Oracle Java 21 (LTS)<br/>
+			</td>
+			
+			<td rowspan="2">Cocoa</td>
+		</tr>
+ 	    <tr class="c0">
+			<td rowspan="1">
+				12<br/>
+				13
+			</td>
+			<td rowspan="1">
+				M1 (arm64)
+			</td>
+            <td rowspan="1">
+            	OpenJDK 17.0.8 (LTS)<br/>
+            	OpenJDK 21 (LTS)<br/>
+            </td>
+   		</tr>
+	</table>
+ </center>
+
+<p>As stated above, <i>we expect that Eclipse works fine on other current
+  Java VM and OS versions but we cannot flag these as reference platforms without
+  significant community support for testing them.</i></p>
+<p>With respect to GTK 3 versions: the <i>required</i> version of GTK 3 which Eclipse SDK 4.33
+will run on is 3.22 or newer.
+</p>
+
+<p>With respect to GTK 3 themes: Adwaita theme is guaranteed to work. Eclipse SDK will run with other GTK 3 themes,
+however we cannot flag these as reference versions without significant community support for testing and/or development
+of fixes. Bugs that are reproducible only with themes other than Adwaita will be given a lower priority (or may not be fixed at all),
+compared to bugs which are reproducible on the target environments listed above.
+</p>
+</div>
+
+<p:internationalization>
+<p>The Eclipse SDK is designed as the basis for internationalized products. The
+  user interface elements provided by the Eclipse SDK components, including dialogs
+  and error messages, are externalized. The English strings are provided as the
+  default resource bundles.</p>
+<p>Latin-1, DBCS, and BiDi locales are supported by the Eclipse SDK on all reference platforms.</p>
+<p>The Eclipse SDK supports GB 18030 (level 1), the Chinese code page standard,
+  on Windows, Linux and the Macintosh.</p>
+</p:internationalization>
+
+</p:target_environments>
+ 
+<!-- ============================================== -->
+
+<p:compatibility_with_previous_releases>
+<div>
+
+<h3>Compatibility of Release 4.33 with 4.32</h3>
+<p>Eclipse 4.33 will be compatible with Eclipse 4.32</p>
+
+<p><strong>API Contract Compatibility:</strong> Eclipse SDK 4.33 will be upwards
+  contract-compatible with Eclipse SDK 4.32 except in those areas noted in the
+  <a href="https://help.eclipse.org/2024-09/topic/org.eclipse.platform.doc.isv/porting/eclipse_4_33_porting_guide.html" target="_top">
+    <em>Eclipse 4.33 Plug-in Migration Guide</em>
+  </a>. Programs that use affected APIs and extension points will need to be ported
+  to Eclipse SDK 4.33 APIs. Downward contract compatibility
+  is not supported. There is no guarantee that compliance with Eclipse SDK 4.33
+  APIs would ensure compliance with Eclipse SDK 4.32 APIs. Refer to
+  <a href="https://wiki.eclipse.org/index.php/Evolving_Java-based_APIs">
```

Comment:
https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/Evolving-Java-based-APIs.md is the new home for this document. Please update the link.